From d6568ee7e1f85025725597d1cdcb62b3d0a1ded1 Mon Sep 17 00:00:00 2001
From: David Runge <dave@sleepmap.de>
Date: Mon, 9 Jan 2023 20:09:25 +0100
Subject: [PATCH 1/2] Switch to correct PEP517 build-system

pyproject.toml:
Since poetry is used, switch to the build-system setup as documented
upstream (https://python-poetry.org/docs/pyproject#poetry-and-pep-517).
---
 pyproject.toml | 16 ++--------------
 1 file changed, 2 insertions(+), 14 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 6554128..1ab7617 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -44,17 +44,5 @@ tox = "^tox 3.11"
 
 
 [build-system]
-requires = [
-    "poetry>=0.12",
-    # See https://github.com/pypa/setuptools/issues/2353#issuecomment-683781498
-    # for the rest of these requirements, 
-    # -ETJ 31 December 2020
-    "setuptools>=30.3.0,<50",
-    "wheel",
-    "pytest-runner",
-    "setuptools_scm>=3.3.1",
-
-]
-
-build-backend = "poetry.masonry.api"
-
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
-- 
2.39.0

